frontend/public/components/pod.tsx

```diff
@@ -1015,6 +1016,8 @@ export const PodsPage: React.FC<PodPageProps> = ({
     fieldSelector,
   });
 
+  const error403 = loadError && loadError.code === 403;
```

Comment:
Strictly speaking, this check isn't correct since a user could create pods, but not have permission to list pods. Create and list can be controlled separately through roles.